crates/runtimelib/Cargo.toml

```diff
@@ -32,7 +32,6 @@ glob = "0.3.1"
 base64 = "0.22.0"
 
 [features]
-default = ["tokio-runtime"]
```

Comment:
This is absolutely necessary for using this as a library successfully (especially if a subdependency for another package), so let's ship runtimelib 1.0 with it.